NEWS.md

```diff
@@ -41,6 +41,10 @@
 * in `describe` the specification of custom aggregation is now `function => name`;
   old `name => function` order is now deprecated
   ([#2401](https://github.com/JuliaData/DataFrames.jl/pull/2401))
+* [PrettyTables.jl](https://github.com/ronisbr/PrettyTables.jl) is now the
+  default back-end to print DataFrames to text/plain; the print option
+  `splitcols` was removed and the output format was changed
```

Comment:
is this still true? maybe we have also changed other kwargs (if there are too many to list just say that a lot has changed and give a reference)

Reply:
Only `splitcols` was removed. All others `kwargs` have the same functionality.

Reply:
OK